src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentVerifyRequest.cs

```diff
@@ -0,0 +1,72 @@
+﻿using SwedbankPay.Sdk.PaymentOrders;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.Payments.CardPayments
+{
+    public class CardPaymentVerifyRequest
+    {
+        public CardPaymentVerifyRequest(Operation operation,
+                              CurrencyCode currency,
+                              string description,
+                              string userAgent,
+                              CultureInfo language,
+                              Urls urls,
+                              PayeeInfo payeeInfo,
+                              string payerReference = null,
+                              bool generatePaymentToken = false,
+                              bool generateReccurenceToken = false,
+                              CreditCard creditCard = null)
+        {
+            this.Verify = new CardPaymentVerifyRequestObject(operation, currency, description, payerReference, generatePaymentToken,
+                                               generateReccurenceToken, userAgent, language, urls, payeeInfo,
+                                               creditCard);
+        }
+
+
+        public CardPaymentVerifyRequestObject Verify;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5742807)